interfaces/cython/cantera/transport.pyx

```diff
@@ -261,7 +261,9 @@ cdef class Transport(_SolutionBase):
             return get_transport_1d(self, tran_getThermalDiffCoeffs)
 
     property multi_diff_coeffs:
-        """Multicomponent diffusion coefficients [m^2/s]."""
+        """Multicomponent diffusion coefficients, D_{i,j} [m^2/s].
+        
+        The coefficient D_{row,column} is used for computing the contribution of concentration gradients in species [column] to the diffusive flux of species [row]."""
```

Comment:
I feel like it's less obvious that `row` and `column` are placeholders for particular species indices here than just continuing to use `i` and `j`.

Reply:
You are assuming, then, that all users understand that in `D_{i,j}` we mean that `i = row` and `j = column`?

Reply:
How about:

> The coefficient D_{i,j} is used for computing the contribution 
        of concentration gradients in species ``j`` to the diffusive flux of 
        species ``i``, where ``i = row`` and ``j = column``.

Reply:
Actually, @speth I may be coming around to your point - knowing `row` and `column` is almost irrelevant, since the indices are how you access the individual coefficients, anyway. So regardless of which is row and which is column, `D[i,j]` in python or `D(i,j)` in Matlab is how you access the value (C++ is different, since it is a flattened matrix, but that is already spelled out in the docstring).

I would be fine either with what I suggest in the comment above, or just replacing `D_{i,j}` with `D[i,j]` or `D(i,j)`, as appropriate.